app/models/fleet.rb

```diff
@@ -12,8 +12,12 @@ class Fleet < ApplicationRecord
   delegate :name, to: :unit
   delegate :capacity, to: :unit
 
+  def facility?
```

Comment:
`delegate :name, :capacity, :facility, :weight, to: :unit`

Reply:
done

Reply:
ah, I delegated as :facility?